src/Router/Boot.tsx

```diff
@@ -1,48 +1,55 @@
 import { Theme, themeProps } from "@artsy/palette"
+import { track } from "Analytics"
 import React from "react"
 import { GridThemeProvider } from "styled-bootstrap-grid"
 import { GlobalStyles } from "Styleguide/Elements/GlobalStyles"
 import { Grid } from "Styleguide/Elements/Grid"
 import { BreakpointVisualizer } from "Styleguide/Utils/BreakpointVisualizer"
 import { Provider as StateProvider } from "unstated"
+import Events from "Utils/Events"
 import { ResponsiveProvider } from "Utils/Responsive"
 import { ContextProvider } from "../Components/Artsy"
 import { AppState } from "./state"
 import { BootProps } from "./types"
 
-export const Boot: React.SFC<BootProps> = ({ children, ...props }) => {
-  const appState = new AppState(props)
+// TODO: Do we want to let Force explicitly inject the analytics code?
+@track(null, { dispatch: data => Events.postEvent(data) })
```

Comment:
This is were all the track events end up bubbling-up to and get reported to the host app (Force) through `Events.postEvent()`.

Reply:
👍